app/components/Texting/components/FormView/Form.module.scss

```diff
@@ -27,10 +27,38 @@
   }
 }
 
+.phoneInputBox {
+  display: flex;
+  margin-top: 10px;
+  background-color: $color-white;
+  border: 1px solid #DDDDDD;
+  border-radius: 2px;
+  letter-spacing: 1px;
+  &:active, &:hover, &:focus {
+    box-shadow: 0 2px 4px 0 rgba(0,0,0,0.1);
+  }
+}
+
+.phoneInputPrefix {
+  padding: 0.9375rem 0 0.9375rem 0.9375rem;
+  color: $color-grey6;
+  font-size: 1.0625rem;
+}
+
+.phoneInput {
+  border: none;
+  background: $color-white;
+  padding-left: 5px;
+  margin: 0; /* to avoid Safari for adding extra margin */
```

Comment:
This is a little bit surprising, since I thought our [reset stylesheet](https://github.com/ShelterTechSF/askdarcel-web/blob/7e4103e80cedfd96566c15f0800eac9913c86fc0/app/styles/utils/_reset.scss) would have taken care of cross-browser differences like this. Looking at it more closely, it seems like we created a custom reset stylesheet by mixing together multiple other ones, so it's not a vanilla reset stylesheet, and it has no reset styles for `input` elements.

Could you add a style to that file to reset all `input` element margins to 0? This could potentially have a large effect on the entire site, so we may want to look through the rest of the pages on the site to make sure we don't cause any major issues elsewhere. We may even want to do this as a completely separate PR before this PR merges in, just so we take care of it as a single, self-contained change.